src/angular.prefix

```diff
@@ -3,4 +3,4 @@
  * (c) 2010-2016 Google, Inc. http://angularjs.org
  * License: MIT
  */
-(function(window, document, undefined) {
+(function(window) {
```

Comment:
IOW, shadowning `undefined` with sth else is more or less equivalent to shadowing any other builtin like `Object`, `Array`, `Object.prototype` methods etc. There are lots of ways in which someone can create a broken environment in which Angular breaks. I don't think it buys us much to defend just against shadowing `undefined` and it's impossible to guard against everything.